app/src/main/java/com/globant/practice/presentation/view/fragment/SubscriberDetailsFragment.java

```diff
@@ -0,0 +1,60 @@
+package com.globant.practice.presentation.view.fragment;
+
+import android.app.Fragment;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import com.globant.practice.R;
+import com.globant.practice.domain.model.User;
+
+/**
+ * Initialize the components of the fragment and manage the communication with the presenter
+ */
+public class SubscriberDetailsFragment extends Fragment {
+    private String nickname;
+
+    /**
+     * Returns a new instance of the SubscriberDetailsFragment and sets the user instance
+     *
+     * @param nickname user nickname
+     * @return new SubscriberDetailsFragment instance
+     */
+    public static SubscriberDetailsFragment newInstance(String nickname) {
+        SubscriberDetailsFragment subscriberDetailsFragment = new SubscriberDetailsFragment();
+        subscriberDetailsFragment.setNickname(nickname);
```

Comment:
This is absolutely a bad practice in Android. Attributes are passed to Fragments using Bundle objects through  setArguments.